src/com/rcv/CVRReader.java

```diff
@@ -103,21 +105,36 @@ private static Sheet getFirstSheet(String excelFilePath) {
       String cvrFileName = new File(excelFilePath).getName();
       // cvrIndex for generating cvrIDs
       int cvrIndex = 1;
+      // counts of unrecognized candidate strings
+      Map<String, Integer> unrecognizedCandidateCounts = new HashMap<>();
 
       // Iterate through all rows and create a CastVoteRecord for each row
       while (iterator.hasNext()) {
         // cvr is the object parsed from the row
-        CastVoteRecord cvr = parseRow(iterator.next(), cvrFileName, cvrIndex++);
-        castVoteRecords.add(cvr);
+        try {
+          CastVoteRecord cvr = parseRow(iterator.next(), cvrFileName, cvrIndex++);
+          castVoteRecords.add(cvr);
+        } catch (UnrecognizedCandidateException e) {
+          // increment the count
+          unrecognizedCandidateCounts.merge(e.getName(), 1, Integer::sum);
```

Comment:
Smooth! Can we add a test config to trigger this error? I was testing it by slightly modifying an existing config (i.e. just deleting out a few candidates).